src/presentational/HelpUsImplementThis.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Tooltip from 'material-ui/Tooltip';
+import './HelpUsImplementThis.css';
+
+const HelpUsImplementThis = ({
+  children,
+  featureDescriptionUrl = 'https://waffle.io/codefordenver/members'
+}) => {
+  return (
+    <Tooltip
+      className="HelpUsImplementThis"
+      title="Help us implement this feature"
+    >
+      <a href={featureDescriptionUrl} target="_blank" rel="noopener noreferrer">
```

Comment:
@wdoug what does `noreferrer ` attribute do here?

Reply:
This is cool! So when you click on the tooltip, does it actually direct the developer to the `featureDescriptionUrl`? Could people start to overwrite this default url to a specific issue on github per the component?

Reply:
Exactly @sungbae